Gems/WhiteBox/Code/Source/SubComponentModes/EditorWhiteBoxTransformMode.h

```diff
@@ -52,12 +68,33 @@ namespace WhiteBox
             const IntersectionAndRenderData& renderData,
             const AzFramework::ViewportInfo& viewportInfo,
             AzFramework::DebugDisplayRequests& debugDisplay);
+        void SetTransformMode(WhiteBox::TransformType type);
+
         bool HandleMouseInteraction(
             const AzToolsFramework::ViewportInteraction::MouseInteractionEvent& mouseInteraction,
             const AZ::EntityComponentIdPair& entityComponentIdPair,
             const AZStd::optional<EdgeIntersection>& edgeIntersection,
             const AZStd::optional<PolygonIntersection>& polygonIntersection,
             const AZStd::optional<VertexIntersection>& vertexIntersection);
+    private: 
+        void CreateTranslationManipulators();
+        void CreateRotationManipulators();
+        void CreateScaleManipulators();
+
+        PolygonSelection refreshPolygonSelection(WhiteBoxMesh* whitebox, const PolygonIntersection& intersection);
```

Comment:
for the moment I'm just going to use this to update the selection. might make sense to move this into a class maybe?

Reply:
Okay sounds good, let me know how you get on 👍